### ISSUE TITLE
Deny access to user with no permissions

### ISSUE THREAD

**Issue by BulatKha** [provectus/kafka-ui#2946]
Might be taken into work after the #753 

### Which version of the app are you running?

[5900f8e](https://github.com/provectus/kafka-ui/commit/5900f8e)

### Is your proposal related to a problem?

User, that can pass the authentication, gets to the page with no clusters on it. Which is a bit strange.

### Describe the solution you'd like

User with no permissions on any cluster, should be declined access instead.

### Describe alternatives you've considered

Separate page with "No permissions provided for the user! Contact your administrator for support" message

**Comment by sarkarshuvojit** [comment]
I would need some help understanding how to replicate this issue. 
I will try to write whatever I understand, please correct me if I'm wrong.

In order for me to replicate the issue,

1. I have to set up the project with RBAC enabled, will refer to [this](https://docs.kafka-ui.provectus.io/configuration/rbac-role-based-access-control)
2. Create a user which has no permissions 
3. Try to open the dashboard

**Comment by Haarolean** [comment]
@sarkarshuvojit yeah, basically, just log in as any user which will have no applicable subjects. Feel free to ping me on discord again if you encounter any problems with that.

**Comment by Haarolean** [comment]
@sarkarshuvojit any updates?

**Comment by volodya953** [comment]
Any updates?

**Comment by Haarolean** [comment]
@volodya953 #4255 -> https://github.com/kafbat/kafka-ui

**Comment by volodya953** [comment]
@Haarolean  Is this feature implemented there or do we need to recreate this topic?

**Comment by Haarolean** [comment]
> @Haarolean  Is this feature implemented there or do we need to recreate this topic?

No it's rather not. Please raise a new issue.
